### PR TITLE
Some bug fixes [nhentai]

### PR DESCRIPTION
**Pull request by Curstantine**
1. Fixed the duplication bug.
2. Added language as a subtitleText for pagedResults
3. API support for `getHomePageSections` and `getViewMoreItems` (psstt no more web scrapping)
4. URLBuilder support.
5. Removed popular-sort

https://curstantine.github.io/h-extensions/ (Outdated)

### CODE REVIEW

**Comment by Curstantine** [comment]
I’m wondering why I never did statics for those helper methods, I’ll change it if needed.

File: src/NHentai/NHentai.ts [hunk 1]
```diff
@@ -15,11 +14,12 @@ import {
...
 
-import { NHENTAI_DOMAIN, QUERY, TYPE, PAGES, capitalize } from "./Functions"
+const NHENTAI_DOMAIN = "https://nhentai.net"
 
 export const NHentaiInfo: SourceInfo = {
-  version: "2.2.1",
+  version: "2.2.2",
   name: "nHentai",
   description: `Extension which pulls 18+ content from nHentai. (Literally all of it. We know why you're here)`,
   author: `VibrantClouds`,
```
Comment (ConradWeiser) [inline src/NHentai/NHentai.ts:25]: You've done such a large overhaul at this point, feel free to slap your name on this instead of mine!

**Review by ConradWeiser — DISMISSED** [review]
Since your `Helper` class is so small, you don't have to worry about it taking up too much memory on the heap. Instead of calling `new Helper().<function>`, why not just make it an instance value and reference that multiple times? 
This is just a nitpick though, not a huge deal.

Some comments on the code structure, nothing dealbreaking. I'll put an approval on this since it's all so minor.

**Comment by Curstantine** [comment]
I'll fix those and push to the same request. Don't merge for now. Thanks!

**Comment by Curstantine** [comment]
1. Moved to static methods.
2. Comments on `getResponse` and `getResponseArray`.

Thanks!

**Comment by Curstantine** [comment]
#65

**Pull request closed without merging**